component/azstorage/block_blob.go

```diff
@@ -666,6 +669,8 @@ func (bb *BlockBlob) getBlobAttr(blobInfo *container.BlobItem) (*internal.ObjAtt
 		Flags:  internal.NewFileBitMap(),
 		MD5:    blobInfo.Properties.ContentMD5,
 		ETag:   sanitizeEtag(blobInfo.Properties.ETag),
+		GID:    ReadMetadata(blobInfo.Metadata, common.POSIXGroupMeta),
```

Comment:
we already parse metadata somewhere to get directory and symlink stuff, ideally these shall be part of those or that function itself shall be optmised to get these fields.